### PR TITLE
Docs/update readme

### PR DESCRIPTION
**Pull request by dcochico**
## What I did:
- Added README

## Did this break anything?
- [x] No
- [ ] Yes

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

## Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [ ] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [ ] The code is DRY. If it's not, I tried my best
- [ ] I reviewed my code before pushing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,66 @@
...
-
-Test
\ No newline at end of file
+# BackYarder
+[DEPLOYMENT LINK](https://backyarder.vercel.app/)
+
+[BACKEND REPO](https://github.com/Backyarder/backyarder-be)
+
+## ABSTRACT
+- BackYarder is an innovative app designed to revolutionize the way gardeners brainstorm and optimize their gardens! This app empowers gardeners of all skill levels to create their dream gardens with ease. Key features include a large database of plants with detailed information, including care instructions, and a drag-and-drop tool to experiment with different arrangements. BackYarder enables the user to create a beautiful garden tailored to their unique vision. Whether you're a seasoned gardener or just starting your gardening journey, this app will be the perfect companion for you!
```
Comment (mbenfowler) [inline README.md:7]: love

File: README.md [hunk 2]
```diff
@@ -1,48 +1,66 @@
...
+- The user can remove unplanted items from their garden.
+![](https://user-images.githubusercontent.com/126428377/265617067-60b845b6-554e-45f5-acab-e49e1f8f36d3.gif)
+- The user can reset their garden and start fresh.
+![](https://user-images.githubusercontent.com/126428377/265617509-f67b13e8-b354-4ec4-8b31-c75325c52f21.gif)
+
+## SET UP
+- Fork this repository.
+- Copy the SSH key in order to clone your forked repository down into your local device.
+- `cd` into the cloned directory.
+- Run `npm install` to install project dependancies.
```
Comment (mbenfowler) [inline README.md:39]: dependencies

File: README.md [hunk 3]
```diff
@@ -1,48 +1,66 @@
...
+- Front End:
+  - Daniel Cochico: [GitHub](https://github.com/dcochico) | [LinkedIn](https://www.linkedin.com/in/daniel-c-3a0201176/)
+  - Matt Fowler: [GitHub](https://github.com/mbenfowler) | [LinkedIn](https://www.linkedin.com/in/mbenfowler/)
+  - Alexa VanSchaardenburg: [GitHub](https://github.com/AlexaVanSchaardenburg) | [LinkedIn](https://www.linkedin.com/in/alexa-r-vanschaardenburg/)
+<br>
+- Back End:
+  - Kim Bergstrom: [GitHub](https://github.com/kbergstrom78) | [LinkedIn](https://www.linkedin.com/in/kimberley-bergstrom/)
+  - Boston Lowrey: [GitHub](https://github.com/BLowrey24) | [LinkedIn](https://www.linkedin.com/in/boston-lowrey/)
+  - Myles Nottingham: [GitHub](https://github.com/MylesNottingham) | [LinkedIn](https://www.linkedin.com/in/mylesnottingham/)
+  - Austin Woods: [GitHub](https://github.com/boomclear) | [LinkedIn](https://www.linkedin.com/in/austin-woods-1830aa195/)
```
Comment (mbenfowler) [inline README.md:53]: the formatting here needs to be fixed up

File: README.md [hunk 4]
```diff
@@ -1,48 +1,66 @@
...
+
+## LEARNING GOALS
+- Explore and implement TypeScript concepts that have not been explicitly taught while at Turing
+- Gain experience using continuous integration tools to build and automate the deployment of features
+- Focus on communication between front-end and back-end teams
+
+## FUTURE EXTENSIONS
+- The user should receive timely alerts for watering plants in their garden.
+- The user should be able to input their location and be shown what hardiness zone they are in.
+- The user should be able to customize the dimensions of their garden grid.
```
Comment (mbenfowler) [inline README.md:66]: i'd put something around individual user login.
Also I'd love to put here something along the lines of a user having multiple plots they can manage!

**Review by mbenfowler — CHANGES_REQUESTED** [review]
Just a few notes!

**Review by AlexaVanSchaardenburg — APPROVED** [review]
I love the abstract, very cute! I also love the format and how well you showcased the app

**Review by mbenfowler — APPROVED** [review]

**Pull request merged**